CSS/main-style.css

```diff
@@ -0,0 +1,166 @@
+@font-face {
+  font-family: bold-Lato;
+  src: url(/fonts/Lato2OFLWeb/Lato/fonts/Lato-Black.ttf);
+}
+
+@font-face {
+  font-family: light-Lato;
+  src: url(/fonts/Lato2OFLWeb/Lato/fonts/Lato-Medium.ttf);
+}
+
+@font-face {
+  font-family: "Cocogoose-trial";
+  src: url(/fonts/Lato2OFLWeb/Lato/fonts/cocogoose_trial.otf);
+}
+
+@media only screen and (max-width: 768px) {
+  * {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+  }
+
+  .t-head {
+    padding: 20px 20px;
+  }
+
+  .m-head {
+    line-height: 10px;
+    color: #ec5242;
+    font-size: 21px;
+  }
+
+  .t-head h1 {
+    line-height: 40px;
+    font-family: 'Cocogosse-trial', sans-serif;
+    background-image: linear-gradient(to right, #7a170c, #3b1d11, #ec5141, #ec5141, #ec5141, #ec5141);
+      /* stylelint-disable */
+    -webkit-background-clip: text;
+    background-clip: text;
+    -moz-background-clip: text;
+  /* stylelint-enable */
```

Comment:
**[REQUIRED CHANGES]**

- [ ] Hey, as explained and suggested by the previous review in this [review comments](https://github.com/GhostEsso/Conference-page/pull/1#discussion_r1144585948), Please, remove the stylint disable.
To fix this error, kindly remove all those lines as you only need the ` -webkit-background-clip: text;` rule to achieve the result you want. check your design, everything should work fine with no linter errors. I hope this helps.